insights/specs/sos_archive.py

```diff
@@ -52,6 +52,7 @@ class SosSpecs(Specs):
     cpu_vulns_spec_store_bypass = simple_file("sys/devices/system/cpu/vulnerabilities/spec_store_bypass")
     cpuinfo_max_freq = simple_file("/sys/devices/system/cpu/cpu0/cpufreq/cpuinfo_max_freq")
     cpupower_frequency_info = simple_file("sos_commands/processor/cpupower_frequency-info")
+    crio_conf = glob_file([r"etc/crio/crio.conf", r"etc/crio/crio.conf.d/*-conmon.conf"])
```

Comment:
Why here only read the `*-conmon.conf` from the `/etc/crio/crio.conf.d/` dir? If I understand correctly, you should read all the `.conf` file in this dir.

Reply:
I updated it to "etc/crio/crio.conf.d/*-conmon.conf", will you help me review it.